src/custom/state/claim/hooks/utils.ts

```diff
@@ -177,8 +177,8 @@ export function calculateInvestmentAmounts(
     return {}
   }
 
-  return { vCowAmount: price.quote(amount), investmentCost: amount }
   const amount = CurrencyAmount.fromRawAmount(investCurrency, investedAmount)
+  return { vCowAmount: price.invert().quote(amount), investmentCost: amount }
```

Comment:
but what is the difference between inverting above and now here?

Reply:
is the math found in the method used on line`975` fixing this? if so, damn, that's sketchy af for us to be using this math lib...